src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -120,6 +122,8 @@ class LocalBackend extends Backend with LazyLogging {
     } else {
       if (rc == 0) {
         evaluateCallOutputs(workflowDescriptor, call, hostAbsoluteFilePath, localEngineFunctions, scopedLookupFunction, interpolateStrings=true)
+      } else if (rc == 143) {
```

Comment:
It's only used here? I hate making things a symbolic constant for the sake of making them a symbolic constant - if the argument is that it's unclear WTF it is (I'd agree w/ that in this case) put a comment right here.